packages/stark-ui/src/modules/dialogs/components/prompt-dialog-theme.scss

```diff
@@ -0,0 +1,6 @@
+.stark-prompt-dialog {
+  .button-ok,
+  .button-cancel {
+    color: mat-color($primary-palette, 500);
```

Comment:
Why not apply this with the color property on the buttons?

Reply:
Well, I didn't want to force the colors by using the `color` directive. I use the CSS approach instead so that it is easier to override if needed.